### PR TITLE
Add tx decoding section

### PR DESCRIPTION
**Pull request by lukasschor**

### CODE REVIEW

File: docs/sdks_03_safe_apps.md [hunk 1]
```diff
@@ -71,6 +71,11 @@ While we allow interacting with Safe Apps hosted on a regular web server through
 
 Gnosis runs its own IPFS node and will make sure to pin your Safe App to guarantee availability.
 
+
+#### 4) Help us decode your Safe App transactions
+
+We want to display interactions with Safe Apps as humand-readable as possible. To do this, we need the contract ABI of the contracts that your Safe App interacts with. Please provide us with the ABIs as JSON files, so we can implement transaction decoding for your Safe App interactions.
```
Comment (tschubotz) [inline docs/sdks_03_safe_apps.md:77]: How can we be sure that the ABIs are correct? Do we double check that? Or should we rather ask for addresses of used verified contracts on Etherscan where we can then pull the ABI from? :)

Comment (lukasschor) [reply docs/sdks_03_safe_apps.md:77]: @rmeissner @Uxio0 What would be the preferred process to receive the ABI from Safe App developers to decode their Safe App interactions?

Comment (Uxio0) [reply docs/sdks_03_safe_apps.md:77]: Most of the ABIs were pulled from etherscan, that way works for me and we make sure we are supporting the same version that is deployed. I would prefer etherscan (or sourcify)

Comment (rmeissner) [reply docs/sdks_03_safe_apps.md:77]: Agree with @Uxio0 that sourcify would be the preferred way, but in the end you cannot manipulate the abi that much (only parameter names)

**Pull request merged**